CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 ## Submitting an Issue
 We use the [GitHub issue tracker](https://github.com/cloudfoundry-community/stackdriver-tools/issues) to track bugs and features.
-Before submitting a bug report or feature request, check to make sure it hasn't already been submitted. You can indicate
+Before submitting a bug report or feature request, check to make sure it has not already been submitted. You can indicate
```

Comment:
If you're changing hasn't -> has not, consider making this sentence:

> Before submitting a bug report or feature request, check to make sure it has not been submitted already.

or 

> Before submitting a bug report or feature request, check to make sure it has not yet been submitted.

(negative sentence gets yet vs already)